media/css/tabzilla/tabzilla.less

```diff
@@ -599,7 +599,7 @@ html[dir="rtl"] {
     height: 150px;
     color: #fff;
     background-color: #95368c;
-    background-image: url(/media/img/home/voices/promos/gear/gear-high-res.png);
+    background-image: url(/media/img/tabzilla/gear-high-res.png);
```

Comment:
It looks like the old tabzilla gear store promo was referencing an image from the old home page (yikes). I moved the image to a different directory.